src/seedu/addressbook/AddressBook.java

```diff
@@ -199,15 +199,23 @@
      * ====================================================================
      */
     public static void main(String[] args) {
+        initialiseProgram(args);
+        while (true) {
+            runProgram();
+        }
+    }
+
+    private static void runProgram() {
```

Comment:
This method name can be improved, right now it is very ambiguous. maybe change its name to represent what the code within the method does